plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -584,19 +636,77 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/user_status/api/v[0-9]+/hea
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
     
+# while trying to update the Weather location
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/weather_status/api/v[0-9]+/location" \
+    "id:9508802,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+# When trying to update user status message
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/user_status/api/v[0-9]+/user_status/message" \
+    "id:9508803,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE'"
+
 
 #
 # [ Users ]
 #
 
-# Allow users to be deleted
+# Allow users to be deleted and user profile settings to be updated
 SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/cloud/users/" \
     "id:9508850,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE'"
+
+# Allow users Profile visibility settings to be updated in Personal info
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/profile" \
+    "id:9508851,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+
+# [ Survey Client ]
+#
+
+# FP when trying to reject the survey client
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/survey_client/api" \
+    "id:9508860,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+#
+# [ Mobile App ]
+#
+
+# FP when trying to remove account from Mobile App
+SecRule REQUEST_FILENAME "@contains /ocs/v2.php/core/apppassword" \
```

Comment:
Did you also want to use a regular expression here like in the other rules?

Reply:
I also thought about using `@rx` but then decided not to. I'll leave this as is.

Reply:
Okay, I realized what's wrong and changed it. Thanks.